com.is_slick.rmac.rmac.yml

```diff
@@ -0,0 +1,30 @@
+id: com.is_slick.rmac.rmac
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rmac
+finish-args:
+  - --filesystem=host
```

Comment:
Why host?

Reply:
I don't necessarily know where users store their code, so I just wanted to give them more liberty.

Reply:
home would be fine, I doubt anyone is storing code in `/opt` or in external drives. It's better to use the least privileged permission first.